### DISCUSSION TITLE
Images Not Working on Live

### DISCUSSION THREAD

**Discussion by thepapisogram** [vercel/next.js#71961] [Help] [answered]
### Summary

Hello. I used <Image> and on my local development server, the image shows. However on my live server, the image does not show and I get a fetch error 502. A workaround I used was to add the unoptimized option to the <Image> component. I am using next js 15. Could someone please explain what happened and how i can use my image without the unoptimized option. Below is the image component
```
<Image
   className="block"
   src="/logo.png"
   width={70}
   height={70}
   alt="Logo"
   unoptimized
/>
```
There is a logo.png file in the public file of my project. And the image shows on local server

**Comment by icyJoseph** [comment]
How is this project deployed? Is it using export mode?

**Reply by thepapisogram** [reply]
It's deployed on netlify. Yes it's using export. The type in the package.json is module

**Reply by icyJoseph** [reply]
Interesting... Image optimizations, are not supported in export mode, https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features

Could that be the issue?

**Comment by aniruddhaadak80** [comment, marked as answer]
Hi @thepapisogram ,

The `502 fetch error` on live for images that work locally in Next.js 15 could be due to the following:

1. **Image Optimization Settings**: In production, Next.js optimizes images by default. If the server configuration doesn’t support it, the image may fail to load. 

2. **Unoptimized Fix**: Using `unoptimized` bypasses Next.js's optimization, which works locally but is less efficient for production.

### Solution:
Check your server or CDN configuration to ensure it supports Next.js image optimization, or consider using an external image optimization service if needed.

**Reply by thepapisogram** [reply]
Thank you very much. I'll try that and let you know how it goes
